src/python/pants/backend/scala/bsp/rules.py

```diff
@@ -151,11 +172,35 @@ async def collect_thirdparty_modules(
         for target in applicable_lockfile_entries.values()
     )
 
-    resolve_digest = await Get(Digest, MergeDigests(cpe.digest for cpe in classpath_entries))
+    digests = []
+    for cpe in classpath_entries:
+        digests.append(cpe.digest)
+    for alse in applicable_lockfile_source_entries:
+        new_file = FileEntry(alse.file_name, alse.file_digest)
+        digest = await Get(Digest, CreateDigest([new_file]))
```

Comment:
I'm aware all these individual `CreateDigest` etc would not be done one-by-one in a for-loop, I just left everything as simple as I could to present the overall approach

Reply:
Then I suggest a TODO comment here as a reminder to just convert this to a single `CreateDigest` with a list of `FileEntry`.